grpc-api/server/native/src/main/resources/grpc-api-native.prop

```diff
@@ -1 +1,5 @@
 includefiles=grpc-api.prop
+
+jpy.jpyLib=/usr/local/lib/python3.7/dist-packages/jpy.cpython-37m-x86_64-linux-gnu.so
+jpy.jdlLib=/usr/local/lib/python3.7/dist-packages/jdl.cpython-37m-x86_64-linux-gnu.so
+jpy.pythonLib=/usr/lib/x86_64-linux-gnu/libpython3.7m.so.1.0
```

Comment:
If this file now exclusively only works inside docker, it should be renamed and moved. By design, the -native and -docker projects have different configs (though these are the only substantial changes, the name of this file, the contents, and the reference to it from the bootstrap properties), and this change should only exist on the docker side.

Perhaps remove this file for native and add it only when docker/server copies sources from this project, likewise with its own bootstrap (or command line arg sysprop to pass to the docker startup)? Or set these sysprops from docker command line?

Reply:
I think we'll want to completely re-do some bootstrap and properties stuff if we need to extract this out - right now, there isn't a good extensible way for packagers to add their own configuration on top. Potentially related to https://github.com/deephaven/deephaven-core/discussions/1105

Reply:
Fixed